.yarn/versions/61ef8dac.yml

```diff
@@ -0,0 +1,3 @@
+releases:
+  "@yarnpkg/plugin-npm": minor
+  "@yarnpkg/plugin-npm-cli": minor
```

Comment:
I used `yarn version check --interactive` here.

These are the two packages I touched. I selected `unsure` for all the other packages that depend on these.
Should they be patch/minor releases? If not, do I need to explicitly list all of them here as `declined`?

Reply:
Minor for those two is correct! I think the only other required is `@yarnpkg/cli` which should be "minor" as well. For the rest, select "declined" as we won't need to release them for this feature.

Reply:
Will update. Thanks 🙏